src/main/resources/data/pilot-courts.csv

```diff
@@ -6,4 +6,13 @@ GUILDFORD,GU14PS,2020-02-24,2020-02-24
 LINCOLN,LN57PS,2020-03-09,2020-03-09
 CENTRAL LONDON,WC2A2LL,2020-03-02,2020-03-02
 CARDIFF,CF101ET,2020-03-16,2020-03-16
-
+NORWICH,NR31UR,2020-04-14,2020-04-14
+CHELMSFORD,CM20PP,2020-04-14,2020-04-14
+WANDSWORTH,SW152SU,2020-04-14,2020-04-14
+BROMLEY,BR13PX,2020-04-14,2020-04-14
+WATFORD,WD171EZ,2020-04-14,2020-04-14
+LEICESTER,LE16HG,2020-04-14,2020-04-14
+PORTSMOUTH,PO12EB,2020-04-14,2020-04-14
+KINGSTONUPONHULL,HU12EZ,2020-04-14,2020-04-14
```

Comment:
We have `CENTRAL LONDON` as two words; guess this should be spaced too

Reply:
Fixed in commit 2